<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUser</class>
 <widget class="QDialog" name="EditUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editing user N</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>id</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="text">
      <string>Set new password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="text">
      <string>group</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="text">
      <string>Grif</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>File access label</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>Folder Access label</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="listWidget">
     <item>
      <property name="text">
       <string>Granted file access</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QListWidget" name="listWidget_2">
     <item>
      <property name="text">
       <string>Granted Folder access</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
